### PR TITLE
Add several basic policies for the 2i2c hubs

### PR DESCRIPTION
**Pull request by choldgraf**
This adds three policies to our infrastructure documentation. We've discussed each of them over time but hadn't yet put them in a public place. I believe that we still need to improve them all, and in some cases may need a lawyer to give an official stamp of approval, so for now I have noted each of them as non-legally-binder policies that are there to set expectations.

Unclear if we can close out any of these issues so I'll cross-ref them below:

- ref: https://github.com/2i2c-org/infrastructure/issues/342
- ref: https://github.com/2i2c-org/docs/issues/139
- ref: https://github.com/2i2c-org/team-compass/issues/388

### CODE REVIEW

File: policy/privacy.md
Comment (sgibson91) [inline policy/privacy.md:20]: Maybe link to the definition of Community Representative here

File: policy/acceptable-use.md
Comment (sgibson91) [inline policy/acceptable-use.md:32]: I would like us to explicitly mention crypto-mining as an activity that contravenes our acceptable use policy. I think we allude to it with "automated transaction-based commands" - but I would like to be crystal clear that we will not look kindly upon users found crypto-mining

**Comment by choldgraf** [comment]
Both comments addressed - thanks @sgibson91

**Comment by sgibson91** [comment]
I haven't given an explicit approving review btw, because I think more eyes from the team would be good here. But if one doesn't come in within an appropriate timeframe, 👍🏻 from me to merge.

**Comment by choldgraf** [comment]
I've also just now pinged CS&S to take a look at these docs, since they might be needed for any "official / legal" policies that we have. So let's give it a few days and re-visit next week if nobody has provided another review since then. @sgibson91 if you "approve" I will still hold off until then before merging or moving forward.

**Review by sgibson91 — APPROVED** [review]

**Comment by choldgraf** [comment]
FYI I heard from Paige at CS&S that they're OK with us posting this now if we wish. I've also asked if they can take a closer look but they said we can disentangle making these public vs. making them "legally official"

**Comment by choldgraf** [comment]
OK gonna merge this one in so that we have something to point people to! We can iterate from there

**Pull request merged**
